### PR TITLE
Use leaderboard API to GET and POST scores into leaderboard app

### PR DESCRIPTION
**Pull request by yodit93**
### In this project, I have

- [ ] created a new game using leaderboard API.
- [ ] implemented a click event to get data from the API.
- [ ] implemented a submit event to the form to post data to the API.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -15,4 +14,8 @@ form.addEventListener('submit', (e) => {
   addList(namevalue, scorevalue);
   inputName.value = '';
```
Comment (emmiiorji) [inline src/index.js]: Your project looks great. Kindly ensure that user data reflects on the scoreboard when saved. You seem not to be updating the DOM after saving the task.

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @yodit93, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<div align='center'>

</div>

## Good Highlights✔️

- Submit button bits the correct API
- The refresh button gets score data
- Good use of asyns and await with the fetch method to send and receive data


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (yodit93) [reply src/index.js]: Hello @emmiiorji 
Thank you for your suggestion.
Please check the project's instructions. According to that, the DOM is updated using the refresh button when it fetches data from the API. The submit button is for saving the data into API.

**Review by topeogunleye — APPROVED** [review]
### Hi @yodit93, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and description :heavy_check_mark:
 
## [Optional] suggestions:
* Nothing to mention.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**